src/index.html

```diff
@@ -11,8 +14,57 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute is pointing to an SCSS file. Browsers cannot directly interpret SCSS files; they require CSS files. Make sure to compile your SCSS to CSS and link the resulting CSS file instead.